src/web/src/routes.js

```diff
@@ -48,6 +62,11 @@ var router = new VueRouter({
       component: EditSemestersPage,
       name: "EditSemesters",
     },
+    {
+      path: "/admin/manageaccounts",
+      component: ManageUserAccountPage,
+      name: "ManageAccounts",
+    },
```

Comment:
Please remove this Manage User Account section for now, you can stash the changes or create another branch with the code.

Besides that should be good for merge

Reply:
Thank you, I already removed all sections associated with Manage Account